README.md

```diff
@@ -85,24 +85,24 @@ tfswitch #will automatically switch to terraform version 0.14.4
 3. Hit **Enter** to install.
 ### Install latest implicit version for stable releases
 1. Install the latest implicit stable version.
-2. Ex: `tfswitch -s 0.13` or `tfswitch --latest-stable 0.13` downloads 0.13.6 (latest) version.
+2. Ex: `tfswitch -s 0.13` or `tfswitch --latest-stable 0.13` downloads <1.1.0, >=1.0.0, 1.0 downloads <2.0.0, >=1.0.0.
 3. Hit **Enter** to install.
 ### Install latest implicit version for beta, alpha and release candidates(rc)
 1. Install the latest implicit pre-release version.
-2. Ex: `tfswitch -p 0.13` or `tfswitch --latest-pre 0.13` downloads 0.13.0-rc1 (latest) version.
+2. Ex: `tfswitch -p 0.13` or `tfswitch --latest-pre 0.13` downloads 0.13.0-rc1.
 3. Hit **Enter** to install.
 ### Show latest version only
 1. Just show what the latest version is.
 2. Run `tfswitch -U` or `tfswitch --show-latest`
-3. Hit **Enter** to show.
+3. Hit **Enter** to print.
 ### Show latest implicit version for stable releases
 1. Show the latest implicit stable version.
-2. Ex: `tfswitch -S 0.13` or `tfswitch --show-latest-stable 0.13` shows 0.13.6 (latest) version.
-3. Hit **Enter** to show.
+2. Ex: `tfswitch -S 0.13` or `tfswitch --show-latest-stable 0.13` prints 0.15.5
```

Comment:
I'm sure they won't release any more version lower than 1.0, but just to stay on safe side:
```suggestion
2. Ex: `tfswitch -S 0.13` or `tfswitch --show-latest-stable 0.13` prints 0.15.5 (latest version which falls under `>=1.0.0, <1.1.0` constraint as of June, 2022)
```